cmd/commandfuncs.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build linux
```

Comment:
The build tag, i.e. `// +build linux` , means the code within this file is only readable and available when the target is Linux, which is why the build is failing for all other platform. Look at the CI results on the "Checks" tab above.

Reply:
@gdhameeja you can see an example of how to do this properly with the `sigtrap.go`, `sigtrap_nonposix.go` and `sigtrap_posix.go` files in the root of the project.

You can't set a build tag on a file that is needed on _all_ platforms. For this feature, you'll need to abstract out the logic that you run to platform-specific files, and have a do-nothing stub for unsupported platforms.